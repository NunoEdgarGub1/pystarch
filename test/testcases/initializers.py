x01 = [1, 2, 3]
x02 = [1, 'a', 2]
x03 = {1, 2, 3}
x04 = {1, 'a', 2}
x05 = {'a': 1, 'b': 2}
x06 = {'a': 1, 2: 'b'}
