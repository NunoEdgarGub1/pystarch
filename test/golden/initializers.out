x01 List(Num)
x02 List(Num)
x03 Set(Num)
x04 Set(Num)
x05 Dict(Str,Num)
x06 Dict(Str,Num)

testcases/initializers.py:2 inconsitent-types "List" (Num, Str, Num)
testcases/initializers.py:4 inconsitent-types "Set" (Num, Str, Num)
testcases/initializers.py:6 inconsitent-types "Dict" (Str, Num)
testcases/initializers.py:6 inconsitent-types "Dict" (Num, Str)
